src/main/java/oshi/software/os/linux/LinuxFileSystem.java

```diff
@@ -71,10 +114,19 @@
             // FileStore toString starts with path, then a space, then name in
             // parentheses e.g., "/ (/dev/sda1)" and "/proc (proc)"
             String path = store.toString().replace(" (" + store.name() + ")", "");
-            // Exclude special directories
-            if (path.startsWith("/proc") || path.startsWith("/sys") || path.startsWith("/run") || path.equals("/dev")
-                    || path.equals("/dev/pts"))
-                continue;
+
+            // Exclude pseudo file systems
+            if (this.pseudofs.contains(store.name())) {

```

Comment:
Two things stopping me from merging this is this conditional. One: I don't understand what it's doing or why.  It may just need some comments to explain what's going on, but the three nested if statements are also confusing (could be a single if with "or" conditions?